tests/bluetooth/gatt/prj.conf

```diff
@@ -0,0 +1,11 @@
+CONFIG_TEST=y
+CONFIG_ZTEST=y
+
+CONFIG_BT=y
+CONFIG_BT_CTLR=n
+CONFIG_BT_NO_DRIVER=y
+
+CONFIG_BT_DEBUG_LOG=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_GATT_DYNAMIC_DB=y
+
```

Comment:
@Vudentz could you fix this trailing whitespace at EOF (the extra empty line), and then we could merge this.

Reply:
Fixed.